tarantool/mesh_connection.py

```diff
@@ -24,8 +26,68 @@ def getnext(self):
         self.pos = (self.pos + 1) % len(self.addrs)
         return self.addrs[tmp]
 
+def parse_uri(uri_str):
```

Comment:
I suggest to use flake8 to check the code style of a newly added code against PEP8.

Reply:
Seem ok

Reply:
$ flake8 tarantool/mesh_connection.py | wc -l
14

Reply:
Validation reworked. Warnings added